roles/monitoring/tasks/cluster-monitoring-common.yaml

```diff
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 ---
-- name: cluster-monitoring - create userdir
+- name: Cluster-monitoring - create userdir
```

Comment:
It is a task and play naming rule for Ansible-lint:

name[casing] - All names should start with an uppercase letter for languages that support it.

The same rule drove changes for OSS to remediate Ansible-lint rule violations there https://github.com/sassoftware/viya4-iac-k8s/pull/78/commits/efe6cb7d08c29a62022c09967990d4bff23e01d7

Jay made the suggestion to change `Tls` to `TLS`, which was done so your suggestion to use `V4M` instead of `V4m` would keep us consistent there.
I'll search for other tasks that start with acronyms and uppercase all letters of those as well.

Reply:
Any variable or reference in the name to a variable should keep its case. All there should be addressed.

Reply:
Are the uppercase task names for values like "cluster-monitoring" where that is the accepted casing for in that case the tag name the main concern, or is it a global concern with uppercasing any of the task/play names?

Reply:
@thpang, I've restored the case in a new set of commits, could you pull this PR, review those changes and alert me to any that I might have missed?